### DISCUSSION TITLE
We need move .chia dir to another disk,not default in disk C.It's growing too fast.

### DISCUSSION THREAD

**Discussion by mapleaj** [Chia-Network/chia-blockchain#7543] [Ideas]
Rigth now is like this:
blockchain_v1_mainnet.sqlite 7.66GiB
.chia is totle 10.8GiB
And my space disk only 6GiB in disk C.
Most of people do not have enough space in disk C. How do we move C:\Users\Username.chia to another disk? 
In the future it's may more than 1TiB,did you image it? Now we need move this dir to another disk.It's growing too fast.

**Comment by SlugFiller** [comment]
This has been discussed in Issues. You need to set an environment variable called CHIA_ROOT (defaults to "~/.chia/mainnet"). Setting environment variables varies by OS. Assuming Windows (since "disk C" isn't a particularly cross-platform term), it would be Start->"Edit the system environment variables" -> "Environment Variables..." -> "System variables" -> "New..." -> "Variable name: CHIA_ROOT" "Variable value: D:\chia\mainnet" -> OK -> OK.

After that, you will need to close the client, manually move the "mainnet" folder, and run the client again.

**Reply by mapleaj** [reply]
Got it. Thank you. It's better to add this to README

**Comment by rmica** [comment]
Can't symbolic links help?
Move .chia to a bigger disk partition \<new-chia-path\>.
Then cd to original directory and link it.
LInux: ln -s \<new-chia-path\> .chia
Windows10: mklink /D \<new-chia-path\> .chia

**Reply by darvd29** [reply]
Thank you!
